tests/test_cart.py

```diff
@@ -684,6 +685,9 @@ def test_cart_fourth_step_requires_auth(db, client):
 
 @override_settings(STRIPE_PUBLISHABLE_KEY='pk_fake')
 def test_cart_fourth_step_renders_correctly(db, user_client):
+    if not settings.STRIPE_SECRET_KEY:
+        # Skip test if no Stripe key is available
+        pytest.skip('Skipping cart test because Stripe keys are not set')
```

Comment:
you can also use 

```python
@pytest.mark.skipif(not settings.STRIP_SECRET_KEY, reason='Skiping cart test because Stripe keys are not set')
def test_cart_fourth_step_renders_correctly(db, user_client):
    ...

```

Reply:
Interesting. I used the approach from the pytest documentation.